lib/service/service.go

```diff
@@ -4124,7 +4124,11 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 				trace.Component: component,
 			})
 
-			log.Infof("Starting Kube proxy on %v.", cfg.Proxy.Kube.ListenAddr.Addr)
+			kubeListenAddr := fmt.Sprintf("%v", listeners.kube.Addr())
```

Comment:
Does Addr have a string method you can call instead of using Sprintf?

Reply:
Updated to use String() where available